redhat-cop/uncontained.io
# Ensure footer fills empty space at the bottom of a page

#### What is this PR About?
Fixes styling of footer to force it to fill any empty space at the bottom of the page on large screens.

Also fixes the openshift buildConfig to build the main fork/master branch.

#### How should we test or review this PR?

Build site:
```
npm install
npm start
```

Then look on the homepage at high resolution. You should see the blue footer bar fill the available horizontal space rather than "floating" above empty gray space.

#### Is there a relevant Trello card or Github issue open for this?
n/a

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

-----


-----

**Please note: we may close your PR without comment if you do not check the boxes above and provide ALL requested information.**
